tests/log_parser/test_log_view_artifact_builder.py

```diff
@@ -36,60 +36,12 @@ def do_test(log):
     assert act == exp
 
 
-def test_crashtest_passing():
```

Comment:
This has a lot of failing tests because of logs with steps and the expected json that still contains step_data. I started cleaning up this file (thinking they were mostly outdated) but was perhaps a bit overzealous. It looks like @armenzg  updated a few test cases for the tests with the "old log parser" comments this year when I checked `git blame`. It probably makes sense to update these logs so they don't reference logs with steps, but I'm not sure which to keep and which to delete.

Reply:
If I just updated the expected json for each test case (to remove steps), I think these tests would still pass. But the question is, do we need all of these test cases - esp ones referencing the "old log parser"??

Reply:
I've checked the blame and I believe I only applied black formatting automatically.
I don't actually know this part of the code base.

Reply:
I suggest skipping the tests so Travis completes and submits the results to codecov.
By doing so, you can determine which part of the code they actually exercise.
It might be that after removing steps data functionality these tests become irrelevant.

Reply:
That's a really good suggestion. I'll do that.

Reply:
Early on in the project, I tended to add way too much test data.  So I'm sure we don't need all of it, if there seems like a lot.  Sometimes regenerating it from the new code and then changing the `assert`s to match the new ids/etc is a better approach.  That way we're sure we're testing the new structure accurately.  Some of this data may be ancient BuildBot output from older test harnesses, etc...

Reply:
It does appear to mostly just be testing the different regexes/cases in the ErrorParser. I've enabled enough to make sure we don't have a dip in coverage, but for the rest that was skipped I'd like to clean it up in a separate pr if that's OK.